src/main/java/frc/robot/commands/DriveCommands.java

```diff
@@ -40,6 +41,7 @@ public void initialize() {
         pid.setSetpoint(0); // Degrees from target.
         pid.setTolerance(1/* degree(s) */);
         pid.enableContinuousInput(-180, 180); // Degrees.
+        flywheel.start(); // Will get turned of in Shoot command.
```

Comment:
If we exit the aiming drive mode without shooting, we should turn off the flywheel.

Reply:
Depending on how the driver wants the buttons mapped, we _could_ exit the aiming command before the shoot command is started, Not sure how to handle that...

Reply:
The flywheel's default command could turn it off.

Reply:
Same problem. If the Aim and Shoot are separate commands, and there is any delay between aim finishing and shoot starting, then the default command will kick on and will stop the flywheel.

Maybe we could give the default command a delay. So that if there are no commands on the flywheel for X seconds, then stop it.

Reply:
Seems like the question is whether we need to allow shooting without aiming. If we don't allow that, we could bind the shoot command to require the aim and shoot buttons both be pressed. So the driver would aim and while aiming, shoot.

Reply:
I think we would want to give more freedom to how we map the commands to buttons. And instead have a default command handle any cleanup. I added a a TODO.